app/components/admin/allowed_table_actions_component.rb

```diff
@@ -15,6 +15,6 @@ def actions
     end
 
     def table_actions_component
-      @table_actions_component ||= Admin::TableActionsComponent.new(record, **options.merge(actions: actions))
+      @table_actions_component ||= Admin::TableActionsComponent.new(record, actions: actions, **options)
```

Comment:
With this change, the code would be the same as `**({ actions: actions }.merge(options))`, which is the opposite of what we had :thinking:.

```suggestion
       @table_actions_component ||= Admin::TableActionsComponent.new(record, **options, actions: actions)
```